app/lib/config.js

```diff
@@ -1,9 +1,17 @@
+function getRule() {
```

Comment:
I'm not sure `getRule` is the best name for this. Perhaps `getSchedule` would be more accurate? And as there is some logic going on I think it would be better suited to live in its own module/file.